ajax/libs/peaks.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "peaks.js",
+  "version": "0.6.0",
+  "description": "Frontend app for displaying audio waveforms",
+  "homepage": "http://waveform.prototyping.bbc.co.uk/",
```

Comment:
@pvnr0082t I was able to access the homepage yesterday. Unable to access it right now. I think this is a temporary problem from their side.